phangsPipeline/casaCubeRoutines.py

```diff
@@ -120,19 +182,94 @@ def copy_mask(infile, outfile, huge_cube_workaround=True):
     Copy a mask from infile to outfile. Includes a switch for large cubes, where getchunk/putchunk can segfault
     """
 
-    if huge_cube_workaround:
-        os.system('rm -rf ' + outfile + '/mask0')
-        os.system('cp -r ' + infile + '/mask0' + ' ' + outfile + '/mask0')
+    #if huge_cube_workaround:
+    #    os.system('rm -rf ' + outfile + '/mask0')
+    #    os.system('cp -r ' + infile + '/mask0' + ' ' + outfile + '/mask0')
+    #else:
+    #    myia = au.createCasaTool(casaStuff.iatool)
+    #    myia.open(infile)
+    #    mask = myia.getchunk(getmask=True)
+    #    myia.close()
+    #
+    #    myia = au.createCasaTool(casaStuff.iatool)
+    #    myia.open(outfile)
+    #    mask = myia.putregion(pixelmask=mask)
+    #    myia.close()
+    
+    mask = get_mask(infile)
+    
+    # use putregion to update pixel mask
+    
+    myia = au.createCasaTool(casaStuff.iatool)
+    
+    myia.open(outfile)
```

Comment:
The shape check could also go in a helper function to make it easier to track the ia open/close calls.